dace/codegen/targets/cuda.py

```diff
@@ -134,12 +137,60 @@ def _emit_sync(self, codestream: CodeIOStream):
             DACE_CUDA_CHECK({backend}DeviceSynchronize());'''.format(backend=self.backend))
 
     def preprocess(self, sdfg: SDFG) -> None:
+        # Find GPU<->GPU strided copies that cannot be represented by a single copy command
```

Comment:
I moved the transformation into the code generation because it really is the final step before generating the copy code. Access node -> access node patterns appear in SDFGs and should be natively supported by the representation for as long as we could keep them. The copy-to-map transformation makes the schedule explicit and may inhibit further transformations such as redundant array removal.